rest-api/etl/setup_database_for_etl.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -ae
+
+# Prepares databases in Cloud SQL for running the RDR -> OMOP ETL. 
+# Imports "cdm" and "voc" databases located in GCS, which were produced by running 
+# setup_local_database_for_etl.
+# If the databases already exist, drop them prior to running this.
+# Note: this takes a while. Go get some coffee while it's running!
+
+USAGE="tools/setup_database_for_etl.sh --project <PROJECT> --account <ACCOUNT>"
+while true; do
+  case "$1" in
+    --account) ACCOUNT=$2; shift 2;;
+    --project) PROJECT=$2; shift 2;;
+    -- ) shift; break ;;
+    * ) break ;;
+  esac
+done
+
+if [ -z "${ACCOUNT}" ]
+then
+  echo "Usage: $USAGE"
+  exit 1
+fi
+
+if [ -z "${PROJECT}" ]
+then
+  echo "Usage: $USAGE"
+  exit 1
+fi
+CREDS_ACCOUNT=${ACCOUNT}
+
+echo "Activating service account..."
+source tools/auth_setup.sh
+gcloud auth activate-service-account circle-deploy@all-of-us-rdr-staging.iam.gserviceaccount.com --key-file=$CREDS_FILE
```

Comment:
Can we avoid naming `circle-deploy@all-of-us-rdr-staging.iam.gserviceaccount.com` in this (otherwise generic) script? This would appear to tie us to the staging env.

Reply:
The vocabulary files are in a GCS bucket under pmi-drc-api-test (which this service account has access to.) We're going to use those files independent of what environment we're updating... the vocabulary doesn't change from environment to environment.

Reply:
So I think we're saying "vocab always lives in the test project" --  two notes

1. Referring to `gs://all-of-us-rdr-vocabulary` makes sense, but why do we need to hard-code a reference to the project in which this bucket is housed?

2. Can't we just give access to this bucket to SAs as needed? So if we're working on ETL in staging, we'd ensure the staging SA has access to the `gs://all-of-us-rdr-vocabulary` bucket, but the code here could be generic?

Reply:
(And one more note: if we plan to flow vocab data into prod, it may be best to store our vocab data in an environment with more prod-like access control sensibilities and auditing; might create a new project for this.)

Reply:
Yeah, changed setup_database_for_etl.sh to use service account for project, added comment for why we use test in setup_local_database_for_etl.sh.

Re: other note. OK. I'll think about this as a follow-up.